luarules/gadgets/team_transfer/api_gadgets.lua

```diff
@@ -0,0 +1,138 @@
+---@meta
```

Comment:
jeeeze is this canonical in LUA? Is there no way to put this crap in a separate file? I really hate this.